packages/flutter/lib/src/material/button.dart

```diff
@@ -75,9 +76,14 @@ class RawMaterialButton extends StatefulWidget {
 
   /// Called when the button is tapped or otherwise activated.
   ///
-  /// If this is set to null, the button will be disabled, see [enabled].
+  /// If this and [onLongPress] are set to null, the button will be disabled, see [enabled].
```

Comment:
If this callback and [onLongPress] are null, then this button will be disabled.

Here and elsewhere: to add a link a related method, include a "See also" section:
```
///
/// See also:
///
///  * [enabled], which is true if the button is enabled.
```